api/v1beta1/aerospikecluster_types.go

```diff
@@ -62,6 +62,8 @@ type AerospikeClusterSpec struct {
 	PodSpec AerospikePodSpec `json:"podSpec,omitempty"`
 	// SeedsFinderServices describes services which are used for seeding Aerospike nodes.
 	SeedsFinderServices SeedsFinderServices `json:"seedsFinderServices,omitempty"`
+	// SecurityContext that will be added to aerospike-server container created by operator.
+	SecurityContext *corev1.SecurityContext `json:"securityContext,omitempty"`
```

Comment:
I think we can put SecurityContext inside PodSpec

Reply:
There is no validation for SecurityContext. According to your use case, do you want to allow the update of SecurityContext by doing a rolling restart or disallow any change?

Reply:
As discussed offline keeping this field here since it relates to AerospikeContainer only but not to the whole pod. renamed it to AerospikeSecurityContext.

I don't see any reasons to forbid updating it. Thus no need for validation, however supported restarting upon changing in SecurityContext.